### ISSUE TITLE
fee_account not listed in Transaction Object details

### ISSUE THREAD

**Issue by marceloneil** [stellar/new-docs#270]
Seems like the `fee_account` field is not listed in the transaction object details in the new docs: https://developers.stellar.org/api/resources/transactions/object/

It is in the old docs though: https://www.stellar.org/developers/horizon/reference/resources/transaction.html

**Comment by lukerQuant** [comment]
Yes I noticed this issue as well. Also some of the descriptions of the asset_type, asset_code & asset_issuer get mixed up in the operations docs. E.g. https://developers.stellar.org/api/resources/operations/object/allow-trust/ has the description of code and issuer the opposite way around

**Comment by lukerQuant** [comment]
@marceloneil do you happen to know if fee_account is optional? The fact that it is in the transaction object example https://developers.stellar.org/api/resources/transactions/object/ but it is in the submit a transaction example https://developers.stellar.org/api/resources/transactions/post/ suggests to me it is optional?
